testsuite/modern-testsuite/src/test/java/org/sonatype/nexus/testsuite/maven/conditional/MavenConditionalRequestIT.java

```diff
@@ -163,7 +147,50 @@ public void conditionalGet() throws Exception {
   }
 
   @Test
+  public void conditionalGetViaGroup() throws Exception {
+    // redirect maven-central
```

Comment:
This duplicates code from above. Factor this out? Also, this seems to unnecessarily couple the test to the default repositories. Why not make a new repository (possibly with a unique name for each test, to avoid test contamination?)